packages/account-v2/src/modules/src/FinancialAssessment/FinancialAssessmentForm.tsx

```diff
@@ -111,64 +109,50 @@ export const FinancialAssessmentForm = () => {
-                {({ dirty, isSubmitting, isValid }) => (
-                    <Form>
-                        {isEUCountry && isFinancialInformationNotComplete && !isFinancialAssessmentUpdateSuccess && (
-                            <InlineMessage type='filled' variant='warning'>
-                                {isMobile
-                                    ? 'To enable withdrawals, please complete your financial assessment.'
-                                    : 'You can only make deposits at the moment. To enable withdrawals, please complete your financial assessment.'}
-                            </InlineMessage>
-                        )}
-                        <div className='flex flex-col w-full min-h-screen space-y-16 lg:w-auto'>
-                            <div className='m-0 overflow-y-auto'>
-                                <div className='flex mb-12 h-24 gap-8 self-stretch lg:self-auto justify-center items-center lg:gap-[11px]'>
-                                    <Text weight='bold'>Financial information</Text>
-                                    <Text size='xs'>(All fields are required)</Text>
-                                    <div className='w-full h-1 flex-[1_1_0] bg-solid-grey-2 lg:flex-shrink-0' />
-                                </div>
-                                <FinancialAssessmentFields />
-                            </div>
-                            <div className='sticky bottom-0 flex justify-end flex-shrink-0 w-full px-24 py-16 border-solid bg-solid-slate-0 border-t-1 border-solid-grey-2'>
-                                {financialAssessmentUpdateError && (
-                                    <InlineMessage type='filled' variant='error'>
-                                        {financialAssessmentUpdateError.error.message}
-                                    </InlineMessage>
-                                )}
-                                {isMobile && !isEUCountry && (
-                                    <Text align='center' size='xs'>
-                                        All fields are required
-                                    </Text>
-                                )}
-                                <Button
-                                    disabled={isSubmitting || !isValid || !dirty}
-                                    isFullWidth={isMobile}
-                                    isLoading={isFinancialAssessmentUpdating}
-                                    size='lg'
-                                    type='submit'
-                                >
-                                    Submit
-                                </Button>
+        <Formik enableReinitialize initialValues={initialValues} onSubmit={handleFormSubmit}>
+            {({ dirty, isSubmitting, isValid }) => (
+                <Form>
+                    <LeaveConfirm />
+                    {isEUCountry && isFinancialInformationNotComplete && !isFinancialAssessmentUpdateSuccess && (
```

Comment:
I don't know if its possible or not here. 
Can we move this condition before return.
Like `const variableName = isEUCountry && isFinancialInformationNotComplete && !isFinancialAssessmentUpdateSuccess`

This might be good for readability.

Reply:
Updated.